content/docs/04-clusters/02-data-center/01-maas/10-register-manage-maas-cloud-accounts.md

```diff
@@ -17,7 +17,10 @@ When you install the Private Cloud Gateway (PCG), a cloud account is auto-create
 
 # Prerequisites
 
-- An installed PCG. Review [Install and Manage MAAS Gateway](/clusters/data-center/maas/install-manage-maas-pcg) for guidance.
+- An installed PCG if you do not have a direct connection to the MAAS environment. 
+
+  If are self-hosting Palette and have a direct connection to the MAAS environment, you can select **Use System Private Gateway**. To learn more about when you would use Palette's PCG or the System Private Gateway, refer to the table in the [Architecture](/clusters/data-center/maas/architecture) guide.
```

Comment:
Let's not refer to the table but rather the PCG section you will add in the architecture page.

Reply:
Sure.